app/views/pages/home.html.erb

```diff
@@ -37,13 +37,16 @@
         <h2 class="display-5">Read the documentation</h2>
         <ul class="list-group lead align-left">
           <li class="list-group-item border-0">
-            <%= link_to 'API documentation', api_path, class: 'text-muted' %>
+            <%= link_to 'Creating and managing API tokens', 'https://github.com/pod4lib/aggregator/wiki/Creating-and-managing-API-tokens', class: 'text-muted' %>
+          </li>
+          <li class="list-group-item border-0">
+            <%= link_to 'Data requirements', 'https://github.com/pod4lib/aggregator/wiki/Data-requirements', class: 'text-muted' %>
           </li>
           <li class="list-group-item border-0">
-            <%= link_to 'Data guidelines', pages_path(id: 'data'), class: 'text-muted' %>
+            <%= link_to 'Uploading data using the API', 'https://github.com/pod4lib/aggregator/wiki/Uploading-data-using-the-API', class: 'text-muted' %>
           </li>
           <li class="list-group-item border-0">
-            <%= link_to 'Data retention policy', pages_path(id: 'retention'), class: 'text-muted' %>
+            <%= link_to 'More...', 'https://github.com/pod4lib/aggregator/wiki', class: 'text-muted' %>
```

Comment:
links with text like "more..." or "click here" are a little weird for screen readers because they might get read in a list of links out of context. this one is tricky though...maybe it could just say "Wiki", idk

Reply:
This should be resolved in f4e3dea.